docs/guide/writing-tasks.md

```diff
@@ -431,6 +432,17 @@ a relation between an encrypted variable and a repository for which the encrypte
     the variable by themselves and editing both the encrypted variable and repo-owner condition
     in the `.cirrus.yml` file.
 
+## Cron Builds
+
+It is possible to configure invocations of re-occurring builds via a well-known Cron expressions. Cron builds can be
+configured on a repository's settings page instead of configuring them in `.cirrus.yml`.
```

Comment:
```suggestion
configured in a repository's settings page, or in the repository's `.cirrus.yml`.
```

Reply:
There is no way to configure Cron Builds in `.cirrus.yml`

Reply:
the way it was worded made it sound like it was possible